glean-core/src/metrics/datetime.rs

```diff
@@ -108,6 +108,21 @@ impl DatetimeMetric {
         glean.storage().record(&self.meta, &value)
     }
 
+    /// Set the metric to the current date/time, including the local timezone offset.
+    ///
+    /// ## Arguments:
+    ///
+    /// * `glean` - the Glean instance this metric belongs to.
+    pub fn set_now(&self, glean: &Glean) {
```

Comment:
I don't feel strongly, but would this be better as `set(&self, glean: &Glean, value: Option<DateTime<FixedOffset>>)` to more closely hew to the Kotlin API?  I'm mainly thinking ahead to when we have a proper public Rust API and we want to document it easily in parallel with the Kotlin API.